gems/pending/VixDiskLib/VixDiskLib.rb

```diff
@@ -96,7 +96,7 @@ def self.setup_env
     vars_to_clear.each do |key|
       my_env.delete(key)
     end
-    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"] + ":" + VIXDISKLIB_PATH
+    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"].to_s + ":" + VIXDISKLIB_PATH
```

Comment:
@matthewd good catch, verified locally. Updated patch to omit the ":" if LD_LIBRARY_PATH isn't set